app/views/anonymous_feedback/problem_reports/index.html.erb

```diff
@@ -0,0 +1,88 @@
+<%= content_for :page_title, "Review Feedback" %>
+<%= content_for :header, "Review Feedback" %>
+
+<%= paginate @feedback, theme: 'twitter-bootstrap-3' %>
+
+<div>
+  <p>Reports rows highlighted in green have already been reviewed.</p>
```

Comment:
This is only true if viewing with "include reviewed" checked, correct? Is there a way we could reflect that in when we show this message?